### PR TITLE
More tweaks to Demo: Bioconductor single cell activity

### PR DESCRIPTION
**Pull request by avahoffman**
Changes:
- add to `_bookdown.yml` so that the page gets rendered
- add learning objectives
- tweak the prereqs slightly (I think it would help to know what expression is and have at least seen some R code before)
- add bit about `sessionInfo()`
- add screenshots and directions for panel exploration and column plot -- pulled screenshots from [here](https://docs.google.com/presentation/d/18J1x5rkWJisiv3EOMVZOma52XWxAVKANTg0Q2FXbOWU/edit?usp=sharing)
- add screenshots and directions for LYZ exploration and violin plot -- pulled screenshots from [here](https://docs.google.com/presentation/d/18J1x5rkWJisiv3EOMVZOma52XWxAVKANTg0Q2FXbOWU/edit?usp=sharing)

Related TODO:
- [x] Update RStudio shutdown directions in AnVIL_Template
     - https://github.com/jhudsl/AnVIL_Template/pull/206

### CODE REVIEW

**Comment by avahoffman** [comment]
@ehumph I think this is ready. Want to give a quick look to make sure nothing is left off? Your script on the slides made this very straightforward -- thank you!! 😄

File: 03-single-cell-with-bioconductor.Rmd [hunk 1]
```diff
@@ -87,15 +91,6 @@ cow::borrow_chapter(
...
-```{r, echo = FALSE, results='asis'}
-cow::borrow_chapter(
-  doc_path = "child/_child_rstudio_tour.Rmd",
-  repo_name = "jhudsl/AnVIL_Template"
-)
-```
-
 # Exercises {#single-cell-with-bioconductor-exercises}
 
 The following exercises will walk you through the process of using iSEE to explore single cell RNA-seq data. You will use data that have already been prepared and is saved in a SummarizedExperiment object. You will use the `pbmc3k` dataset from the `TENxPBMCData` package, which contains gene expression profiles for 2,700 single peripheral blood mononuclear cells. Information on how the data were preprocessed can be found [here](https://isee.github.io/iSEEWorkshopEuroBioc2020/articles/dataset.html).
```
Comment (ehumph) [inline 03-single-cell-with-bioconductor.Rmd]: I think it's actually a SingleCellExperiment object - I made a mistake before.

File: 03-single-cell-with-bioconductor.Rmd [hunk 2]
```diff
@@ -139,17 +134,145 @@ library(iSEE)
 
 First we will take a look at the interactive plots that iSEE can display.
```
Comment (ehumph) [inline 03-single-cell-with-bioconductor.Rmd:136]: Do we need to say anything about the interactive plots? This section feels a little barebones, but I know we've limited time for the actual presentation.

**Review by ehumph — APPROVED** [review]
Two small comments, but otherwise lgtm! Thanks for finishing this (:

File: 03-single-cell-with-bioconductor.Rmd (hunk 2 above)
Comment (avahoffman) [reply 03-single-cell-with-bioconductor.Rmd:136]: I was thinking we'd just scroll around on the dashboard. Do we want to do more than that?

File: 03-single-cell-with-bioconductor.Rmd (hunk 1 above)
Comment (avahoffman) [inline 03-single-cell-with-bioconductor.Rmd]:
```suggestion
The following exercises will walk you through the process of using iSEE to explore single cell RNA-seq data. You will use data that have already been prepared and is saved in a SingleCellExperiment object. You will use the `pbmc3k` dataset from the `TENxPBMCData` package, which contains gene expression profiles for 2,700 single peripheral blood mononuclear cells. Information on how the data were preprocessed can be found [here](https://isee.github.io/iSEEWorkshopEuroBioc2020/articles/dataset.html).
```

File: 03-single-cell-with-bioconductor.Rmd (hunk 2 above)
Comment (ehumph) [reply 03-single-cell-with-bioconductor.Rmd:136]: No, that makes sense to me.

**Pull request merged**